File: Bin/PLDemos/Data/Scripts/ScriptsScene.js
//[-------------------------------------------------------]
//[ Global variables                                      ]
//[-------------------------------------------------------]
var g_Factor = 1;


//[-------------------------------------------------------]
//[ Global functions                                      ]
//[-------------------------------------------------------]
function Update()
{
	PL.Log.OutputAlways("Always");
	PL.Log.OutputCritical("Critical");
	PL.Log.OutputError("Error");
	PL.Log.OutputWarning("Warning");
	PL.Log.OutputInfo("Info");
	PL.Log.OutputDebug("Debug");
}

function getFactor()
{
	return PL.Timing.GetTimeDifference() + PL.Timing.GetFramesPerSecond();
}

File: Bin/PLDemos/Data/Scripts/ScriptsScene.py
#[-------------------------------------------------------]
#[ Global variables                                      ]
#[-------------------------------------------------------]
g_Factor = 1


#[-------------------------------------------------------]
#[ Global functions                                      ]
#[-------------------------------------------------------]
def Update():
	PL['Log']['OutputAlways']("Always")
	PL['Log']['OutputCritical']("Critical")
	PL['Log']['OutputError']("Error")
	PL['Log']['OutputWarning']("Warning")
	PL['Log']['OutputInfo']("Info")
	PL['Log']['OutputDebug']("Debug")
	return

def getFactor():
	return PL['Timing']['GetTimeDifference']() + PL['Timing']['GetFramesPerSecond']()
